core/src/main/java/jenkins/model/ModifiableTopLevelItemGroup.java

```diff
@@ -66,6 +66,7 @@
 
     /**
      * Creates a new job.
+     *
```

Comment:
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format

"Insert a blank comment line between the description and the list of tags, as shown."

The code base is inconsistent with usage, but cocktail napkining with some grep + perl says ~410/676 leading at-tags DO have the blank line as advised.